File: test/fixtures/output/node/request/query.js
var request = require("request");

request({
  "method": "GET",
  "url": "http://mockbin.com/har",
  "qs": {
    "foo": [
      "bar",
      "baz"
    ],
    "baz": "abc",
    "key": "value"
  }
}, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: test/fixtures/output/csharp/httpclient/query.cs
var client = new HttpClient();
var request = new HttpRequestMessage
{
    Method = HttpMethod.Get,
    RequestUri = new Uri("http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value"),
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}
